src/Core/Framework/Struct/AssignArrayTrait.php

```diff
@@ -31,4 +39,126 @@ public function assign(array $options)
 
         return $this;
     }
+
+    /**
+     * @param array<array-key, mixed> $options
+     */
+    private function assignRecursive(array $options): self
+    {
+        foreach ($options as $key => $value) {
+            $propertyName = self::denormalize((string) $key);
```

Comment:
I don't get this step :thinking: is this still a left-over from PayPal? :grin: because we use the same casing in the array keys as the property names are